fix broken nn_convolution test

Camyll opened the pull request:
Summary: Broken by oss diff during oncall by third party contributor

Test Plan: buck test 'fbcode//mode/dev-nosan' fbcode//caffe2/test:nn_convolution -- --run-disabled

Differential Revision: D85899891

atalman reviewed (approved):
lgtm

seemethere reviewed (approved):
Review automatically exported from Phabricator review in Meta.

Camyll committed:
fix broken nn_convolution test (#166666)

Summary:

Broken by oss diff during oncall by third party contributor

Test Plan: buck test 'fbcode//mode/dev-nosan' fbcode//caffe2/test:nn_convolution -- --run-disabled

Reviewed By: seemethere

Differential Revision: D85899891
